src/librustc_mir/transform/validate.rs

```diff
@@ -49,8 +54,25 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
         );
     }
 
-    fn check_bb(&self, location: Location, bb: BasicBlock) {
-        if self.body.basic_blocks().get(bb).is_none() {
+    fn check_bb(&self, location: Location, bb: BasicBlock, edge_kind: EdgeKind) {
```

Comment:
Probably a better name for this function would now be `check_edge`

Reply:
I agree - I’ll update tomorrow